_kbarticles/2021-05-17-troubleshooting-ssh-connections.md

```diff
@@ -19,15 +20,26 @@ Some things to check:
 It is possible your internet service provider or network provider will block traffic on port 2222. You can work around this by using a VPN (like the GSA VPN) or using a native ssh client and following the instructions below.
 
 1. Retrieve the APP_GUID for your app: `cf app APP_NAME --guid`.
-2. Retrive a one-time ssh passcode using `cf ssh-code`. You will be prompted for this code below.
+2. Retrieve a one-time ssh passcode using `cf ssh-code`. You will be prompted for this code below.
 3. Use `ssh` (not `cf ssh`) to connect to your app container. Substitute the value for your APP_GUID: `ssh -p 22 cf:APP_GUID/0@ssh.fr.cloud.gov`. You will be prompted to enter a password. Use the one-time ssh passcode you generated in step 2.
 
 This will open an ssh connection using port 22 to your app instance with index of 0. If you need to connect to a different instance, replace the `0` in the command above.
 
+Note - If this approach does not work for you, and you see an error that says **Permission denied, please try again.**, then you should review the section below on changes to your application in the CAPI.
+
 ### Changes to your app in the CAPI
 
 If you are still experiencing trouble connecting to your app using SSH, it may be the result of some changes in a newer version of the Cloud Foundry API (CAPI). In prior versions of the CAPI, app instances used a single process. In version 3 of the CAPI, there can be more than 1 process that makes up an app. This may occasionally cause issues when trying to SSH to your app. 
 
 If a modification or update was made to an app using a v3 command or process - even if that change was made by someone else (e.g., another member of a development team) - the app’s schema in the Cloud Controller might have changed to version 3, and a standard cf ssh may no longer work. 
 
-If this occurs, you can try using `cf v3-ssh {app-name}`. By default a cf v3-ssh will select the `web` process. You can also select a different process if needed with v3-ssh using the `--process` flag.
\ No newline at end of file
+If this occurs and you are using v6 of the `cf` CLI, you can try using `cf v3-ssh {app-name}`. By default a cf v3-ssh will select the `web` process. You can also select a different process if needed with v3-ssh using the `--process` flag.
```

Comment:
Is it just this part that is distinctive between the V6 and V7 CLIs, or the instructions below as well?  In other words, will the steps that follow this note also work for V7?

If that's the case, highlighting that this statement is for V6 only will help cut down on confusion.  Alternatively, putting the remaining content below under a new sub-heading will also work.

Reply:
Made some changes